app/src/main/java/com/example/respiratorapp/AboutActivity.java

```diff
@@ -26,8 +26,8 @@ protected void onCreate(Bundle savedInstanceState) {
     private ImageView home_btn;
     private ImageView settings_cog;
     protected void initListeners() {
-        home_btn = (ImageView) findViewById(R.id.imageView17);
```

Comment:
These variables should follow camelCase naming conventions [https://www.javatpoint.com/java-naming-conventions](url)